.github/workflows/ci.yml

```diff
@@ -10,6 +10,22 @@ jobs:
   ci:
     runs-on: ubuntu-latest
 
+    env:
```

Comment:
let's create a separate action for it. `ci-e2e.yml.`

```yml
name: Continuous Integration. E2E

env:
  # local ENVs

on:
  push:
    branches:
      - development
  workflow_dispatch:

jobs:
  ci-e2e:
    runs-on: ubuntu-latest

    services:
    #...
```

Reply:
P.S. It's Integration testing